packages/core/database/lib/entity-manager/regular-relations.js

```diff
@@ -279,96 +279,70 @@ const cleanOrderColumnsForInnoDB = async ({
   const { joinTable } = attribute;
   const { joinColumn, inverseJoinColumn, orderColumnName, inverseOrderColumnName } = joinTable;
 
-  const now = new Date().valueOf();
-  const randomHex = randomBytes(16).toString('hex');
+  const randomSuffix = `${new Date().valueOf()}_${randomBytes(16).toString('hex')}`;
```

Comment:
Maybe we can use `crypto.randomUUID()` method ?

Looks like available in v14 (even if MDN says 19? 🤔 )
https://nodejs.org/docs/latest-v14.x/api/crypto.html#crypto_crypto_randomuuid_options

Reply:
Yes maybe it can be used. I would not risk to change it now however, so the PR is merged for the release of tomorrow